src/renderer/chunk_meshing.zig

```diff
@@ -1234,6 +1233,11 @@ pub const ChunkMesh = struct { // MARK: ChunkMesh
 				_ = newBlock.mode().updateData(&newBlock, neighbor, neighborBlocks[neighbor.toInt()]);
 			}
 		}
+
+		if(oldBlock.entityDataClass()) |class| {
+			class.onBreakClient(.{_x, _y, _z}, self.chunk);
```

Comment:
What about the new block?

Maybe better do the missing entity data class methods in a future PR.

Reply:
That was moved by mistake while restoring neighbor updates.

Reply:
Moved back, as for new block, it gets `onPlaceClient` instead in line 1244